README.md

```diff
@@ -293,6 +293,39 @@ This is a "meta" gem, that simply depends on all the others. If you want to
 automatically include support for all languages, you can just include this gem
 and you'll get all you need.
 
+### Private Registry Credential Management
+
+For many ecosystems, Dependabot Core supports private registries. Sometimes this happens by passing the private registry
+credentials directly to the native package managers (`npm`, `pip`, `bundler`, etc), other times it happens within the
+Dependabot Core Ruby code.
+
+```mermaid
+sequenceDiagram
+    Private Registry Credentials->>Dependabot Core: 
```

Comment:
This diagram failed to parse unless these lines had some text after the `:`, which is why there's a trailing space. That'd be fine if i wanted to add text to the arrows, but I'd prefer to keep things simple.

However, my editor wants to strip all trailing whitespace upon save.

I couldn't find an alternative syntax in the mermaid docs for signaling that no text was needed? Only thing I could come up with is the `flowchart LR`, but that wouldn't be consistent with the second diagram where I _do_ want to document the arrows... 🤷‍♂️ 

Any tips from the mermaid experts on how to make this work without a trailing space?

Reply:
Discovered that adding `<br />` works. `&nbsp;` doesn't--it results in adding `"&nbsp;"` as a visible label on the arrows, which is def not what we want.